baqend/chromeUXReports.ts

```diff
@@ -325,31 +326,69 @@ exports.post = async function (db: baqend, req: Request, res: Response) {
           return new db.ChromeUXReportData({
             start: histogram.start,
             density: histogram.density,
-          });
-        });
+          })
+        })
 
       // calculate totalDensity
-      report.totalDensity = calculateTotalDensity(report[metric]);
-      const medianName = MEDIAN_NAME_MAPPING.get(metric) as string;
-      report[medianName] = calculateMedian(report[metric], report.totalDensity);
-    });
+      report.totalDensity = calculateTotalDensity(report[metric])
+      const medianName = MEDIAN_NAME_MAPPING.get(metric) as string
+      report[medianName] = calculateMedian(report[metric], report.totalDensity)
+    })
 
     const isFailed =
       !report.firstContentfulPaint.length ||
       !report.firstPaint.length ||
       !report.domContentLoaded.length ||
       !report.onLoad.length ||
-      !report.ttfb.length;
+      !report.ttfb.length
 
-    report.status = isFailed ? 'FAILED' : 'SUCCESS';
-    await report.save();
+    report.status = isFailed ? 'FAILED' : 'SUCCESS'
+    await report.save()
 
     if (report.device !== 'all') {
-      const normalizedReport = normalizeReport(db, report);
-      await normalizedReport.save();
+      const normalizedReport = normalizeReport(db, report)
+      await normalizedReport.save()
     }
   }
-  res.status(200);
-  res.send({message: 'The queried domain was analyzed successfully.'});
-  return;
-};
+  res.status(200)
+  res.send({message: 'The queried domain was analyzed successfully.'})
+  return
+}
+
+exports.get = async function (db: baqend, req: Request, res: Response) {
+  const param = req.query.url
+  const input = new URL(param)
```

Comment:
Lass uns das auch in dem Try Block hauen. Falls da mal ein string übergeben wird welcher keine URL ist, fliegt hier nämlich ein Fehler.